### PR TITLE
Improve readability of DispatchSelectIf parameterization

### PR DESCRIPTION
**Pull request by bernhardmgruber**
@gonidelis and I got a bit confused tracing parameter names through the code, so here is a small improvement.

### CODE REVIEW

File: cub/cub/device/device_select.cuh [hunk 1]
```diff
@@ -191,16 +191,17 @@ struct DeviceSelect
...
-                       d_in,
-                       d_flags,
-                       d_out,
-                       d_num_selected_out,
-                       SelectOp(),
-                       EqualityOp(),
-                       num_items,
-                       stream);
+      /*KeepRejects*/ false,
+      /*MayAlias*/ false>::Dispatch(d_temp_storage,
```
Comment (elstehle) [inline cub/cub/device/device_select.cuh:195]: How about we add some `constexpr bool` definitions like `constexpr bool keep_reject = false;` instead? Same would apply for similar instantiations of the `Dispatch` template in other places.

**Review by elstehle — APPROVED** [review]
Just a nitpick.

File: cub/cub/device/device_select.cuh (hunk 1 above)
Comment (bernhardmgruber) [reply cub/cub/device/device_select.cuh:195]: I agree. We already have them in the tuning policies anyway.

Comment (bernhardmgruber) [reply cub/cub/device/device_select.cuh:195]: Ha, except that would be a breaking change :D Will add it to the breaking change list.

**Pull request merged**